### PR TITLE
Small video

### PR DESCRIPTION
**Pull request by DarkRaven110**
Add support for changing frame shape

### CODE REVIEW

File: contrib/video_server/video_server.py [hunk 1]
```diff
@@ -166,13 +176,16 @@ def main():
         videos = videos[1:]
     loop_play = args.loop
     fps=args.fps
+    frame_shape = None
+    if args.shape:
+        img_shape = tuple(args.shape)
```
Comment (zmsp) [inline contrib/video_server/video_server.py]: frame shape not getting updated with img_shape

File: contrib/video_server/video_server.py [hunk 2]
```diff
@@ -148,7 +158,7 @@ def main():
         help='Specify a video file path, rtsp stream address or camera value')
     parser.add_argument('-p', '--port', default=6420, type=int)
     parser.add_argument('-a', '--address', default="0.0.0.0")
-
+    parser.add_argument('-s', '--shape', default=None, nargs='+', type=int)
```
Comment (zmsp) [inline contrib/video_server/video_server.py]: might need some help text to make tuple input more apparent

File: contrib/video_server/video_server.py (hunk 1 above)
Comment (DarkRaven110) [reply contrib/video_server/video_server.py]: fixed. I changed variable names for consistency, and forgot one.

File: contrib/video_server/video_server.py (hunk 2 above)
Comment (DarkRaven110) [reply contrib/video_server/video_server.py]: I added a help me.

**Comment by zmsp** [comment]
@mmartial @annieweng @DarkRaven110 added some additional parameters to control the memory size. See adb742f20b4d7a52d362257ba213f94c10f65265

**Comment by zmsp** [comment]
@mmartial @annieweng @DarkRaven110 added some additional parameters to control the memory size. See adb742f Merging with an additional commit on top of Scott's.

**Pull request merged**